backend/apps/events/validators.py

```diff
@@ -0,0 +1,244 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from django.utils.timezone import make_aware
+from utils.helpers.validators import email_validation
+
+
+def time_validation(start_time_input, end_time_input, start_time=None, end_time=None):
```

Comment:
Just to make sure I know what we're restricting here: do we allow events with either only a start time or an end time?